lms/views/dashboard/api/course.py

```diff
@@ -63,6 +64,7 @@ def course_stats(self) -> list[APIAssignment]:
                 APIAssignment(
                     id=assignment.id,
                     title=assignment.title,
+                    course=api_course,
```

Comment:
Here the response becomes a bit verbose because all assignments belong to the same course but that might not be the case as we enable more filtering options.